### DISCUSSION TITLE
[Feature Request] Rewrite CLI using Commander.js

### DISCUSSION THREAD

**Discussion by thecodechef** [stylus/stylus#2853] [Ideas]
**BUGS** in current cli

* safari/webkit in the help command doesn't go where it need to (always brings up a 404 page on apple.com) needs to be switched to the webkit.org reference page
* opera in the help command doesn't go where it need to (always brings up a 404 page on opera) and the website it tells you to go to has been deprecated since 2015
* quirksmode in the help command doesn't go where it need to (always brings up a 404 page on google.com)

**Use Cases**

* Cleaner Code
* Better Readability
* Easier Maintenance
* And of course its made by the creator of Stylus himself

**Comment by thecodechef** [comment]
@iChenLei 
The only reason i'm pinging somebody for this is because no has been talking about it yet and i want some feedback for it that's all

**Comment by iChenLei** [comment]
Hi, @thecodechef  PR welcome
